src/components/users/UserRegisterForm.js

```diff
@@ -54,8 +52,7 @@ class UserRegisterForm extends Component {
             name='email'
             hintText='Email'
             floatingLabelText='Email'
-            validationFn={value =>
-                            !emailValidator.validate(value) && 'Wrong email'}
+            validationFn={value => validateEmail(value)}
```

Comment:
Ask Rėdas to implement API endpoint that checks if email is not already taken. Later it would be nice to implement here server-side validation of it.